avocado/utils/network/exceptions.py

```diff
@@ -0,0 +1,4 @@
+class NWException(Exception):
```

Comment:
Just wondering if it is possible to have a more specific exception here instead os `Exception`.

Reply:
This is something that was already here. I did not create this.

Can you think of a more specific for all the network problems that we have here?

I don't see any problem with this. NWException says that it is our exceptions, and this is good because the user will know that the problem is with Avocado itself.

Reply:
Okay, no problem, I was just wondering :)